CONTRIBUTING.md

```diff
@@ -229,30 +229,7 @@ All the listener classes can be found in `listeners` directory.
 ### Services
 
 All the service classes can be found in `services` directory.
-
-- `CoverageSelectionToggleListener` is a topic interface for showing or hiding highlighting when the coverage is toggled for one test or many tests.
-- `CoverageToolWindowDisplayService` creates the *"Coverage visualisation"* tool window panel and the coverage table to display the test coverage data of the tests generated by EvoSuite.
-- `CoverageVisualisationService` visualises the coverage in the gutter and the editor (by colouring), injects the coverage data into the *"Coverage visualisation"* tool window tab.
-- `ErrorMonitor` class for handling error occurrences.
-- `QuickAccessParametersService` allows to load and get the state of the parameters in the *"Parameters"* tool window panel.
-- `RunnerService` is used to limit TestSpark to generate tests only once at a time.
-- `SettingsApplicationService` stores the application-level settings persistently. It uses `SettingsApplicationState` class for that.
-- `SettingsProjectService` stores the project-level settings persistently. It uses `SettingsProjectState` class for that.
-- `StaticInvalidationService` invalidates the cache statically.
-- `TestCaseCachingService` contains the data structure for caching the generated test cases and is responsible for adding, retrieving and removing (invalidating) the generated tests.
-- `TestCaseDisplayService` displays the tests generated by EvoSuite, in the *"Generated tests"* tool window panel.
-- `TestSparkTelemetryService` sends usage information to Intelligent Collaboration Tools Lab at JetBrains Research if the user has opted in.
-
-### Settings
-
-All the classes related to TestSpark `Settings/Preferences` page can be found in `settings` directory.
-
-- `SettingsApplicationState` is responsible for storing the values of the EvoSuite Settings entries.
-- `SettingsEvoSuiteComponent` displays and captures the changes to the values of the entries in the EvoSuite page of the Settings dialog.
-- `SettingsEvoSuiteConfigurable` allows to configure some EvoSuite settings via the EvoSuite page in the Settings dialog, observes the changes and manages the UI and state.
-- `SettingsPluginComponent` displays and captures the changes to the values of the entries in the TestSpark main page of the Settings dialog.
-- `SettingsPluginConfigurable` allows to configure some Plugin settings via the Plugin page in the Settings dialog, observes the changes and manages the UI and state.
-- `SettingsProjectState` is responsible for storing the values of the Plugin Settings entries.
+We have three services at this point only for managing setings in EvoSuite, LLM-based approach, and the general plugin's settings.
```

Comment:
1. setings -> se**tt**ings
2. **the** general plugin's settings -> general plugin's settings

How about:
"""
We currently have three services for managing EvoSuite settings, the LLM-based approach/generation, and general plugin settings: [_name the services here_].
"""